src/esp/gfx/Renderer.h

```diff
@@ -15,6 +15,7 @@ namespace gfx {
 class Renderer {
  public:
   Renderer();
+  explicit Renderer(bool hasTextures);
```

Comment:
Just a single overload? Would also implicitly document the default state.

```suggestion
  explicit Renderer(bool hasTextures = true);
```

Tho ... from an API usability standpoint, I'd prefer to not have a bool argument and instead do something like

```cpp
enum class Flag {
    NoTextures = 1 << 0,
};

typedef Corrade::Containers::EnumSet<Flag> Flags;
CORRADE_ENUMSET_FRIEND_OPERATORS(Flags)

explicit Renderer(Flags flags = {});
```

because then `Renderer(Renderer::Flag::NoTextures)` is much more descriptive than `Renderer(false)`.

Reply:
Makes sense.